frontends/infinite-corridor/src/pages/FieldPageSkeleton.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react"
+import { BannerPage } from "ol-util"
+import Container from "@mui/material/Container"
+import { useFieldDetails } from "../api/fields"
+
+interface FieldSkeletonProps {
+  children: React.ReactNode
+  name: string
+}
+
+/**
+ * Common structure for field-oriented pages.
+ *
+ * Renders the field title and avatar in a banner.
+ */
+const FieldSkeletonProps: React.FC<FieldSkeletonProps> = ({
+  children,
+  name
+}) => {
+  const field = useFieldDetails(name)
+
+  return (
+    <BannerPage
+      src={field.data?.banner ?? ""}
+      alt=""
+      compactOnMobile
+      bannerContent={
+        <Container className="field-title-container">
+          <div className="field-title-row">
+            <img alt="" src={field.data?.avatar_medium ?? ""} />
```

Comment:
At some point (this or a future PR) this should handle null avatar images better, right now it shows a broken image.  Discussion channels show initials based on title if there is no avatar image, maybe we could do that here too.  Could use a populated alt tag too.

from `open-discussions/src/components/ChannelAvatar`:
```python
          {isDefault ? (
            <div className="avatar-initials">{initials(channel.title)}</div>
          ) : (
            <img
              src={imageUrl}
              alt={`Channel avatar for ${channel.name}`}
              className={`avatar-image`}
            />
          )}
```

Reply:
@mbertrand Good point about the channel title initials. I'll do that in a separate PR.

Re alt tags: I actually removed that on purpose. My understanding is that unless the banner & avatar alt actually add something, they are detrimental. I need to get a functional screenreader, but I believe that on open-discussions, a screen reader would basically read

> Banner image for My Cool Channel. Channel avatar for My Cool Channel. My Cool Channel

banner alt + avatar alt + channel title.... Right now, just reading the channel title seems better.